proposals/action-on-user-settings-changed-api.md

```diff
@@ -0,0 +1,68 @@
+# Extension API Proposal: action.onUserSettingsChanged
+
+## Background
+
+Proposal based on [issue #346](https://github.com/w3c/webextensions/issues/346): New API: UserSettings.isOnToolbar change event
+
+## API Overview
+
+### Objective
+
+Fire an event when user-specified settings relating to an extension's action changed.
+
+### Use Cases
+
+#### Pinning / unpinning extension
+
+Extensions can know if its action is pinned by calling `chrome.action.getUserSettings.` If the extension wants to know when this value changes, it has to poll the API method. Instead, it could register for an event.
```

Comment:
Nit: move `.` (end of sentence) outside of the code block (delimited by backticks).

```suggestion
Extensions can know if its action is pinned by calling `chrome.action.getUserSettings`. If the extension wants to know when this value changes, it has to poll the API method. Instead, it could register for an event.
```

Reply:
Done